Implement system.dns_cache table and dns_cache_max_size server setting (#60257)
ClickHouse/ClickHouse: docs/en/operations/server-configuration-parameters/settings.md

Default: 0

## dns_cache_max_size

Internal DNS cache max size in bytes.

:::note

> We also have reverse cache, so actually memory usage could be twice as much.

Reviewer: ```suggestion
ClickHouse also has a reverse cache, so the actual memory usage could be twice as much.
```
